plugins/channel_header_plug/channel_header_plug.tsx

```diff
@@ -134,7 +151,19 @@ class ChannelHeaderPlug extends React.PureComponent<ChannelHeaderPlugProps, Chan
         this.toggleDropdown(false);
     }
 
+    fireAction = (action: (channel: Channel, channelMember: ChannelMembership) => void) => {
+        if (this.state.disableButtonsClosingRHS) {
+            return;
+        }
+
+        action(this.props.channel, this.props.channelMember);
+    }
+
     fireActionAndClose = (action: (channel: Channel, channelMember: ChannelMembership) => void) => {
+        if (this.state.disableButtonsClosingRHS) {
```

Comment:
Is this necessary? This is for when the buttons are collapsed and you select one element, right?

Reply:
Good call, removed this block